hwdb.d/80-ieee1394-unit-function.hwdb

```diff
@@ -1080,6 +1080,46 @@ ieee1394:ven000002F0mo00000000sp0000A02Dver00000100
   ID_MODEL_FROM_DATABASE=iSweet
   IEEE1394_UNIT_FUNCTION_VIDEO=1
 
+#
+# Digital Everywhere FloppyDTV and FireDtv series.
+#
+
+ieee1394:node:ven0x001287mo0x000024units0x00a02d:0x010001
+ieee1394:ven00001287mo00000024sp0000A02Dver00010001
+  ID_VENDOR_FROM_DATABASE=Digital Everywhere
+  ID_MODEL_FROM_DATABASE=FloppyDTV S/CI
+  IEEE1394_UNIT_FUNCTION_VIDEO=1
```

Comment:
We normally use a single space of indentation for .hwdb files. I think it'd be better to use the same in this file too.
(There is a test for this, but because of some parser issue this is not enforced, see https://github.com/systemd/systemd/pull/18661#issuecomment-781582329.)

Reply:
@keszybz ,

> We normally use a single space of indentation for .hwdb files. I think it'd be better to use the same in this file too.
(There is a test for this, but because of some parser issue this is not enforced, see #18661 (comment).)

I'm OK for the one-space indentation and will prepare revised version for take 2.